docs/en/guides/sre/s3-multi-region.md

```diff
@@ -0,0 +1,381 @@
+
+All three servers must listen for network connections so that they can communicate between the servers and with S3.  By default, ClickHouse listens ony on the loopback address, so this must be changed.  This is configured in `/etc/clickhouse-server/config.d/`.  Here is a sample that configures ClickHouse and ClickHouse Keeper to listen on all IP v4 interfaces.  see the documentation or the default configuration file `/etc/clickhouse/config.xml` for more information.
+
+```xml title="/etc/clickhouse-server/config.d/networking.xml"
+<clickhouse>
+    <listen_host>0.0.0.0</listen_host>
+</clickhouse>
+```
+
+## Start the servers
+
+### Run ClickHouse Keeper
+
+On each Keeper server:
+```bash
+sudo -u clickhouse \
+  clickhouse-keeper -C /etc/clickhouse-keeper/keeper.xml
+```
+
+#### Check ClickHouse Keeper status
+
+Send commands to the ClickHouse Keeper with `netcat`.  For example, `mntr`:
+
+```bash
+echo mntr | nc localhost 9181
+```
+```response
+zk_version	v22.7.2.15-stable-f843089624e8dd3ff7927b8a125cf3a7a769c069
+zk_avg_latency	0
+zk_max_latency	11
+zk_min_latency	0
+zk_packets_received	1783
+zk_packets_sent	1783
+# highlight-start
+zk_num_alive_connections	2
+zk_outstanding_requests	0
+zk_server_state	leader
+# highlight-end
+zk_znode_count	135
+zk_watch_count	8
+zk_ephemerals_count	3
+zk_approximate_data_size	42533
+zk_key_arena_size	28672
+zk_latest_snapshot_size	0
+zk_open_file_descriptor_count	182
+zk_max_file_descriptor_count	18446744073709551615
+# highlight-start
+zk_followers	2
+zk_synced_followers	2
+# highlight-end
+```
+
+### Run ClickHouse Server
+
+On each ClickHouse server run
+```
+sudo service clickhouse-server start
+```
+
+#### Verify ClickHouse Server
+
+When you added the [cluster configuration](#define-a-cluster) a single shard replicated across the two ClickHouse nodes was defined.  In this verification step you will check that the cluster was built when ClickHouse was started, and you will create a replicated table using that cluster.
+- Verify that the cluster exists:
+  ```sql
+  show clusters
+  ```
+  ```response
+  ┌─cluster───────┐
+  │ cluster_1S_2R │
+  └───────────────┘
+  
+  1 row in set. Elapsed: 0.009 sec. `
+  ```
+
+- Create a table in the cluster using the `ReplicatedMergeTree` table engine:
+  ```sql
+  create table trips on cluster 'cluster_1S_2R' (   
+   `trip_id` UInt32,   
+   `pickup_date` Date,   
+   `pickup_datetime` DateTime,   
+   `dropoff_datetime` DateTime,   
+   `pickup_longitude` Float64,   
+   `pickup_latitude` Float64,   
+   `dropoff_longitude` Float64,   
+   `dropoff_latitude` Float64,   
+   `passenger_count` UInt8,   
+   `trip_distance` Float64,   
+   `tip_amount` Float32,   
+   `total_amount` Float32,   
+   `payment_type` Enum8('UNK' = 0, 'CSH' = 1, 'CRE' = 2, 'NOC' = 3, 'DIS' = 4))
+  ENGINE = ReplicatedMergeTree
```

Comment:
Make sure that the `default_replica_path `and `default_replica_name` are defined